### PR TITLE
Added Zapier

### PR DESCRIPTION
**Pull request by woutersamaey**
- Added Zapier to the menu
- Added docs + images to the `setup_deps.sh` script so it pulls and updates automatically

### CODE REVIEW

**Comment by woutersamaey** [comment]
@pavlenex This should be better, right?

**Comment by pavlenex** [comment]
Yup, though the build is failing 
```10:49:09 AM: [vuepress] No matching page found for sidebar item "/Zapier"```

**Comment by woutersamaey** [comment]
@dennisreimann I think I did everything right here, but the build still breaks. Not sure it is related to my changes. I looked at the other examples, but cannot find what's wrong.

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -90,6 +90,7 @@ const sidebarUserGuide = [
     title: 'Integrations',
     collapsable: false,
     children: [
+      ['/Zapier', 'Zapier'],
```
Comment (dennisreimann) [inline docs/.vuepress/config.js:93]:
```suggestion
      ['/Zapier/', 'Zapier'],
```

**Comment by dennisreimann** [comment]
@woutersamaey The build works now, but the link check is failing. With /btcpayserver/zapier#4 merged it'll work again.

**Comment by woutersamaey** [comment]
Hmm, why does it need a trailing slash while all other links do not need it? Bizarre.
I approved the PR btw.

**Comment by woutersamaey** [comment]
@dennisreimann we could further improve the `setup_deps.sh` script by adding `--depth 1` to the other repos. It would make the build faster and reduce disk space + network traffic. I implemented it this way for Zapier. Did you know about this option?

**Comment by woutersamaey** [comment]
@dennisreimann I don't have permissions to re-run the checks I believe. Can you re-run them so we can move forward?

**Comment by dennisreimann** [comment]
> It would make the build faster and reduce disk space + network traffic. I implemented it this way for Zapier. 

Sure, go ahead in a separate PR :)

**Comment by woutersamaey** [comment]
Sure, after this PR as i would be changing the same file again.

**Review by dennisreimann — APPROVED** [review]

**Pull request merged**